static/src/javascripts/projects/commercial/modules/prebid/utils.spec.js

```diff
@@ -237,6 +237,11 @@ describe('Utils', () => {
         expect(shouldIncludePangaea()).toBe(true);
     });
 
+    test('shouldIncludePangaea should return false if section is in blacklist', () => {
```

Comment:
Would it be better to read the exported blacklist and make sure it is false for all the mentioned sections? The property stated is that the blacklist works, but the property tested is that culture is false. They overlap a little now, but aren't quite the same thing.